### ISSUE TITLE
Incompatibility with Emacs 25?

### ISSUE THREAD

**Issue by cpitclaudel** [iqbalansari/emacs-emojify#14]
Hi,

Is the dependence on seq-1.12 right? My emacs (25-master) refuses to install emojify with this message:

```
   Status: Incompatible because it depends on uninstallable packages.
   Archive: melpa

   Version: 20151214.938
   Summary: Display emojis in Emacs
   Requires: seq-1.12 (not available), ht-2.0, emacs-24.3
   Homepage: https://github.com/iqbalansari/emacs-emojify
   Keywords: multimedia convenience 
```

**Comment by iqbalansari** [comment]
Thanks for the bug report. IIRC `seq` will be bundled with Emacs 25, so `package.el` should not have tried fetching it. Let me try this on Emacs master.

**Comment by iqbalansari** [comment]
I think I understand the issue now. `seq` was available on MELPA but was removed in a recent [commit](https://github.com/milkypostman/melpa/commit/76b7b90da2b4a1aef8ccb7aebe6dd335de612d7e), now it is available only on GNU ELPA and the version available is 1.11, I looked only at seq's github repo and assumed that the latest version was 1.12.

I have pushed a fix on master, please try when MELPA builds and report back

Thanks

**Comment by cpitclaudel** [comment]
Thanks for looking into this! I'm still confused though: since I have Emacs 25, I shouldn't have run into this problem anyhow, right?

**Comment by iqbalansari** [comment]
> I'm still confused though: since I have Emacs 25, I shouldn't have run
> into this problem anyhow, right?

Right! I just built current Emacs master and checked, it seems that current master has seq version 1.8 and `emojify` was depending on 1.12 (a higher) version, so `package.el` was trying to fetch the non-existent version (and failing).

I think this has been fixed in the latest build. I should have been more careful, apologies for the breakage.

**Comment by iqbalansari** [comment]
@cpitclaudel did the fix work for you?

**Comment by cpitclaudel** [comment]
No, but I think this is not an issue with this package, actually. Sorry for the trouble. The headers of the seq library say `2.3`, but package sees it as `1.8` in master; I have no idea why. Do you see the same problem on master?

**Comment by iqbalansari** [comment]
No problem. If I understand it correctly you are still not able to install `emojify` due to this missing dependency issue, which odd since seq v1.11 is available on GNU ELPA as such even if the local version of the package is older the newer version should have been installed. In fact I was able to install on Emacs master after the above fix.

> The headers of the seq library say 2.3, but package sees it as 1.8 in master; I have no idea why

I think I know the reason for that. The information about builtin packages is loaded from the file `lisp/finder-inf.el` which is generated when compilation by the Makefile in the `lisp` directory. I think the file is not regenerated on subsequent recompilations, I fixed this by going to the `lisp` directory and manually running the required make task (`make finder-data`).
